.github/workflows/test-google-cloud-platform-install.yaml

```diff
@@ -175,6 +208,28 @@ jobs:
           --quiet \
           ${{ steps.vars.outputs.k8s_cluster_name }}
 
+    - name: Delete the associated NEG (Network Endpoint Groups)
+      if: ${{ always() && steps.helm_install.outcome == 'success' && steps.fetch_neg.outcome == 'success' }}
+      shell: bash
+      run: |
+        delete_neg() {
+            local NEG_NAME="$1"
+            local NEG_ZONES
+            IFS=',' read -r -a NEG_ZONES <<< "$2"
+
+            for NEG_ZONE in "${NEG_ZONES[@]}"
+            do
+              gcloud compute network-endpoint-groups delete \
+                  --project ${{ secrets.GCP_PROJECT_ID }} \
+                  --zone "$NEG_ZONE" \
+                  --quiet \
+                  "$NEG_NAME"
+            done
+        }
+
+        delete_neg "${{ steps.vars.outputs.posthog_events_neg_name }}" "${{ steps.vars.outputs.posthog_events_neg_zones }}"
```

Comment:
non-blocking

I guess these can be single quotes as context substitution happens at workflow gen time not at run time?

Reply:
Fair point, it's just habit :D